pkg/controller/node/nodecontroller.go

```diff
@@ -449,9 +474,8 @@ func (nc *NodeController) monitorNodeStatus() error {
 		// When adding new Nodes we need to check if new zone appeared, and if so add new evictor.
 		zone := utilnode.GetZoneKey(added[i])
 		if _, found := nc.zonePodEvictor[zone]; !found {
-			nc.zonePodEvictor[zone] =
-				NewRateLimitedTimedQueue(
-					flowcontrol.NewTokenBucketRateLimiter(nc.evictionLimiterQPS, evictionRateLimiterBurst))
+			nc.zonePodEvictor[zone] = NewRateLimitedTimedQueue(
```

Comment:
Please avoid strictly formatting changes in PRs as big as this one. It makes reviews even harder.